### PR TITLE
lab-site: Improve various unrelated aspects of the first labs

### PR DESCRIPTION
**Pull request by juliaogris**
* Improve the HSL example to work with drag and flicker less
* Add a video embed
* Fix broken links to docs sites

### CODE REVIEW

File: frontend/lab/samples/intro/welcome.htmlf [hunk 1]
```diff
@@ -8,10 +8,15 @@
...
   That's what we'll build up to in this first lab—a bit of a whirlwind tour!
 </p>
-<p>
-  There is also a step-by-step
-  <a href="https://youtu.be/FBqyiU4iZNY?si=9vFayb_R2rAgEMbX" target="_blank">video guide</a> if you
-  need help..
-</p>
+<p>There is also a step-by-step video guide:</p>
+<iframe
+  src="https://www.youtube.com/embed/FBqyiU4iZNY?si=9vFayb_R2rAgEMbX"
```
Comment (camh-) [inline frontend/lab/samples/intro/welcome.htmlf:13]: Is the `si` query parameter needed? I usually remove query params on youtube links and they work just fine. Or is this some metrics param for tracking watches or something?

**Review by camh- — APPROVED** [review]
:green_book:

File: frontend/lab/samples/intro/welcome.htmlf (hunk 1 above)
Comment (juliaogris) [reply frontend/lab/samples/intro/welcome.htmlf]: yeah, you are right, removed. thank you!

**Review by camh- — APPROVED** [review]
:green_book: still LGTM

**Pull request merged**